### ISSUE TITLE
Documentation is confusing

### ISSUE THREAD

**Issue by stetso** [gideros/gideros#487]
The current state of the documentation is quite confusing to newcomers (like me). There seem to be a couple of guides:

- [API documentation (reference)](http://docs.giderosmobile.com/reference/)
- [Getting started guide](http://docs.giderosmobile.com/)
- [an old (?) wiki](http://giderosmobile.com/DevCenter/index.php/Main_Page)
- [a developers guide](http://giderosmobile.com/guide)
- [the ultimate guide](http://giderosmobile.com/DevCenter/index.php/Ultimate_Guide_to_Gideros_Studio)
- [a FAQ guide](http://giderosmobile.com/DevCenter/index.php/Forum_FAQ)
- [a new wiki](http://wiki.giderosmobile.com/index.php/Welcome!)

On a cursory glance, there seems to be a lot of duplicated information as well as outdated stuff. The main problem is, that the new wiki (which I consider to be the most relevant) is not linked from anywhere on the web page - I could only find it though a link in the forum thread.

I am not sure about the best solution to that, but would help with migrating everything to a single place. Any plans on that?

**Comment by hgy29** [comment]
Yes there was a plan to do that, actually the new Wiki is supposed to be that single place, but oldies are still around until we are sure everything was ported to the Wiki. I think it is the case, but some people don't want me to remove old docs yet. 
There is also a plan to change the old web site to a CMS based system for ease of maintenance, but I personaly lacks the skills to design it.

**Comment by stetso** [comment]
Cool, that sounds good to me. Maybe at some point in the near future the old docs can be removed then.

I was actually thinking about the website as well, as it is quite deterring currently I think. But may I ask why a CMS? There is not much going on (ok, currently it is, but lets be honest, it is not maintaines t all and most of it is not used/outdated anyway) - so what about a static site generator, maybe even using github pages for that!? It would have a few benefits I think: 1) the code would be visible and changeable to everyone via PRs, 2) SSL by default (currently missing from website and forum, so it is not too surprising there are a few hacekd entries) 3) it could be a good opportunity to slim down the site and correct links etc - when everything is documented in the wiki anyway, there would not be much maintenance I think and it would save a lot on running a  PHP-or-whatever-server for the backend of the CMS, 4) the blog could move there too, because Blogger seems outdated as well.

TLDR: I can maybe help with a lightweight design of a new homepage in Jekyll (or something else, but GH-Pages uses it out-of-the-box) but then it would be a static site, no CMS, no logins, just good old HTML ;-)

**Comment by hgy29** [comment]
Idea was to have something that would be community updatable, but yes, something hosted on GH would be fine! Somehow we were also thinking about having a market place (for sell plugins or tools), but that's another topic...

**Comment by stetso** [comment]
Haha, ok, a store would be out of score then (although I am not too sure how useful that would be currently as I cannot guess the size of the userbase...). Well, since I was looking for an excuse to work on a simple webpage anyway, maybe I try to create something an show it in the forum, I would choose Jekyll then as it would make potentially moving it over much easier :)

**Comment by hgy29** [comment]
Since old sites have been shut down, and only the wiki remains, we can consider the issue closed
